### ISSUE TITLE
Project with incorrect number of tasks

### ISSUE THREAD

**Issue by Baconcrisp26** [KaartGroup/tasking-manager#56]
The number of tasks in [this](https://tasks.kaart.com/project/916) project appear to have been calculated incorrectly. The TM says there are only 65, but there are actually 89. No tasks were split in this project.

**Comment by tsmock** [comment]
I don't think we are going to worry about this type of thing until we get around to migrating to TM4. Yes it is an issue, but we have put TM3 on light maintenance mode since TM4 was released. This means that we are only doing bug fixes for _critical_ issues. I.e., entire teams are affected.

And since we still need to port our features over to TM4, I have no clue if this bug affects TM4.
